src/sentry/integrations/utils/metrics.py

```diff
@@ -422,11 +422,25 @@ def get_interaction_type(self) -> str:
 
 
 class IntegrationWebhookEventType(StrEnum):
+    """
+    Provider-agnostic event types for integration webhooks used for metrics tracking.
+
+    Enum names use generic SCM terminology:
+    - "merge request" instead of "pull request" (GitHub) or "merge request" (GitLab)
+    - "CI check" for continuous integration checks (GitHub Check Runs, GitLab Pipelines, etc.)
+
+    String values preserve original GitHub naming for backward compatibility with existing metrics.
+    """
+
+    CI_CHECK = "ci_check"
```

Comment:
We rename the value from `check_run` since we have just implemented this feature in the last two weeks, it's barely used and I don't care if we have a discontinuity somewhere in the GitHub metrics.